### ISSUE TITLE
Spell: Kill command

### ISSUE THREAD

**Issue by roleplay1** [WoWManiaUK/Anachronos#19]
The hunter spell  Kill Command is not working

**Comment by fvicaria** [comment]
Please follow the procedure described below before creating a new issue:
[How to Properly Report an Issue](https://github.com/wow-mania/Redemption/wiki/How-to-Properly-Report-an-Issue)

This will help developers identify and fix the issue quicker. Incorrectly or unclear reports will simply be deleted.

**Comment by flumby** [comment]
Kill command has to be on the player bar not the pet bar. It is a player ability.

**Comment by gamer100** [comment]
it is on the player bar  . it is button 7  on the player bar

**Comment by flumby** [comment]
Yeah kill command just does no damage. No function to it just a cooldown.

**Comment by gamer100** [comment]
an constant cooldown spell is not actif any time
you have to use it to get a cooldown this  spell is  always  on cd and not to use
